sentinel.go

```diff
@@ -169,8 +186,14 @@ func (c *sentinelClient) DoMultiCache(ctx context.Context, multi ...CacheableTTL
 		return nil
 	}
 	attempts := 1
+
+	commands := make([]Completed, 0, len(multi))
+	for _, command := range multi {
+		commands = append(commands, Completed(command.Cmd))
+	}
+	sendToReplica := c.sendAllToReplica(commands)
```

Comment:
Copying the command slice could be too expensive. Could we let `sendAllToReplica` accept an iterator?

Reply:
OK. [13f97ec](https://github.com/redis/rueidis/pull/850/commits/13f97ecb5990a45e02d2c13ca28a7b8396930f05)